src/components/NcButton/NcButton.vue

```diff
@@ -518,7 +683,7 @@ export default {
 					},
 				],
 				attrs: {
-					'aria-label': this.ariaLabel,
+					'aria-label': this.ariaLabel || text,
```

Comment:
`this.ariaLabel` prop was removed, wasn't it?

Reply:
No it is not currently. We use it to show a warning if it is not set :wink:

Reply:
Oh, it was just moved to another place in props